website/content/en/status/report-2023-04-2023-06/caldera.adoc

```diff
@@ -26,4 +26,4 @@ TODO
 
 * Add Caldera testing infrastructure makejail
 * Add FreeBSD to platforms officially supported by MITRE Caldera, see link:https://github.com/mitre/caldera/pull/2752[]
-* Add FreeBSD to platforms officially supported by Red Canary, see link:https://github.com/redcanaryco/atomic-red-team/pull/2450[]
+* Add FreeBSD to platforms officially supported by Red Canary, see link:https://github.com/redcanaryco/atomic-red-team/pull/2450[].
```

Comment:
I remember we look at punctuation in lists every quarter, but I always forget what we decide... We should maybe write it somewhere.
In this case, I would not add this final period, or I would add one at the end of each element of the list: the list does not look as part of a sentence, so I would treat it either as a list of sentences each with its own dot, or as a list of instructions without any dot.

Reply:
> … or I would add one at the end of each element of the list: …

Resolved by [`9adb8e6` (#203)](https://github.com/freebsd/freebsd-doc/pull/203/commits/9adb8e67c6c951aeee45dcfe3e510fdc43dc1139).

----

> … what we decide …

[List item (bullet point) style and punctuation](https://gist.github.com/grahamperrin/9e382d743731d03e275ae0f46aea9649) notes that the previous report (first quarter):

> … included a mixture of styles; uncertainty remained. …

In internationally co-authored content such as status reports, **some mixture is fine** because (I hope) we don't overstep a mark when editing each person's (or group's) contribution; we don't strip too much personality/nationality from writings.